phpBB/styles/prosilver/template/ucp_pm_viewfolder.html

```diff
@@ -1,130 +1,82 @@
-			<select name="mark_option">{S_MARK_OPTIONS}{S_MOVE_MARKED_OPTIONS}</select> <input class="button1 button button-form-bold" type="submit" name="submit_mark" value="{L_GO}" />
-			<div><a href="#" onclick="marklist('viewfolder', 'marked_msg', true); return false;">{L_MARK_ALL}</a> &bull; <a href="#" onclick="marklist('viewfolder', 'marked_msg', false); return false;">{L_UNMARK_ALL}</a></div>
-		</fieldset>
-
-		<hr />
-
-		<div class="action-bar bottom">
-			<!-- INCLUDE display_options.html -->
-			<input type="hidden" name="cur_folder_id" value="{CUR_FOLDER_ID}" />
-
-			<div class="pagination">
-				{TOTAL_MESSAGES}
-				<!-- IF .pagination -->
-					<!-- INCLUDE pagination.html -->
-				<!-- ELSE -->
-					 &bull; {PAGE_NUMBER}
-				<!-- ENDIF -->
+{% include 'ucp_header.html' %}
+
+{% if not PROMPT %}
+	{% include 'ucp_pm_message_header.html' %}
+{% endif %}
+
+{% if messagerow %}
+	<div class="pm">
+		<div class="pm-list-container">
+			<ul class="pm-list">
+				{% for message in messagerow %}
+					<li class="pm-item {{ message.FOLDER_IMG_STYLE == 'pm_unread' ? 'pm-unread' }}">
+						<span class="pm-primary">
+							{% if message.FOLDER_IMG_STYLE == "pm_unread" %}
+								<i class="fa fa-certificate fa-fw" title="Unread message" aria-hidden="true"></i>
+							{% endif %}
+
+							{% if message.PM_CLASS == "pm_marked_colour" %}
+								<i class="fa fa-star fa-fw" title="You marked this message important!" aria-hidden="true"></i>
+							{% endif %}
+
+							<a href="{message.U_VIEW_PM}" title="{message.SUBJECT}" class="pm-title">{message.SUBJECT}</a>
+							<span class="pm-secondary">
+
+								<span class="pm-secondary-item pm-authour">
+									{% if message.PM_CLASS == "pm_foe_colour" %}
+										<i class="fa fa-exclamation-circle fa-fw" title="{message.MESSAGE_AUTHOR} is your foe!" aria-hidden="true"></i>
+									{% else %}
+										<i class="fa fa-user fa-fw" title="{message.MESSAGE_AUTHOR}" aria-hidden="true"></i>
+									{% endif %}
+
+									{% if S_SHOW_RECIPIENTS %}
+										{{ lang('MESSAGE_TO') }} {message.RECIPIENTS}
+									{% else %}
+										{message.MESSAGE_AUTHOR_FULL}
+									{% endif %}
+
+								</span>
+
+								<span class="pm-secondary-item pm-date">
+									<i class="fa fa-clock-o fa-fw" title="{message.SENT_TIME}" aria-hidden="true"></i>
+									{message.SENT_TIME}
+								</span>
+							</span>
+						</span>
+						<span class="pm-meta">
+							<i class="fa fa-square fa-fw pm-checkbox" aria-label="Mark"></i>
+							<input id="mark-1" type="checkbox" hidden>
+						</span>
+					</li>
+				{% endfor %}
+
+			</ul>
+
+		</div><br>
+
+		<div class="pm-viewer">
+			<div class="pm-message">
+				This will be taken care of in another PR
```

Comment:
Why?

Reply:
We decided to have separate PR earlier, but due to delays, I think we should do everything here.